### DISCUSSION TITLE
Please assist to write the code

### DISCUSSION THREAD

**Discussion by ngbeken1monday** [microsoft/Web-Dev-For-Beginners#353] [Q&A]
Write a function (with a name of your choice) that asks for 4 inputs (a, b, c, d) using prompt boxes, converts them to floating point, and prints true in an alert box if (input a x input b) < (input c x input d). Else, print false in an alert box

**Comment by ManuSquall** [comment]
Hello @ngbeken1monday at which point do you encounter difficulties?? 
(If you can give the exercise reference link also please)

If you have done some code, feel free to share it too

**Reply by ngbeken1monday** [reply]
Yes I have done the code

**Comment by dieniceish** [comment]
Trying to learn by watching others. I’m sorry I haven’t contributed anything.

**Comment by Aaron-1974** [comment]
Sorry I'm still learning and so I can't contribute as that's beyond my current abilities, but I'm watching intently.

**Reply by ManuSquall** [reply]
The intention is good! Feel free to contribute anyway, learn by doing is very efficient! Don't focus on mistakes and begins with small things to grab some concept :-)

**Reply by Aaron-1974** [reply]
<scripttypescripttype="text/javascript">  
  
</script>

**Reply by ngbeken1monday** [reply]
<scripttypescripttype="text/javascript">

</scripttypescripttype>

**Reply by ngbeken1monday** [reply]
<scripttypescripttype="text/javascript">

</scripttypescripttype>
